pipelines.adoc

```diff
@@ -9,21 +9,47 @@ permalink: /guides/pipelines-getting-started/
 
 To learn more about pipelines and creating new tasks, see link:https://github.com/tektoncd/pipeline/blob/master/docs/tutorial.md[the pipeline tutorial].
 
-== Tasks and Pipelines
+== Default Tasks and Pipelines
 
-Each application stack contains a set of default tasks and pipelines that are created when the application stack is built. The application stack build process copies the task and pipeline files from link:https://github.com/kabanero-io/collections/tree/master/incubator/common/pipelines/default[the application stack repo] to your `/pipelines` directory.
+The default Kabanero tasks and pipelines are provided in the link:https://github.com/kabanero-io/kabanero-pipelines/tree/master/pipelines/incubator[Kabanero pipelines repository].  These can be associated with an application stack in the Kabanero custom resource definition.  An example is provided below:
+
+```
+apiVersion: kabanero.io/v1alpha1
+kind: Kabanero
+metadata:
+  name: kabanero
+spec:
+  version: "0.6.0"
+  stacks: 
+    repositories: 
+    - name: central
+      https:
+        url: https://github.com/kabanero-io/collections/releases/download/0.5.0/kabanero-index.yaml
+    pipelines:
+    - id: default
+      sha256: 14d59b7ebae113c18fb815c2ccfd8a846c5fbf91d926ae92e0017ca5caf67c95
+      https:
+        url: https://github.com/kabanero-io/kabanero-pipelines/releases/download/0.6.0/default-kabanero-pipelines.tar.gz
+```
+
+When the Kabanero operator activates the CRD, it will associate the pipelines in the pipelines archive with each of the stacks in the stack hub.  The default pipelines are intended to work with all the stacks in the stack hub in the example above. All of the pipeline related resources like the tasks, trigger bindings, and pipelines prefix the name of the resource with the keyword `StackId`.  When the operator activates these, it will replace the keyword with the name of the stack it is activating.
+
+=== Updating Default Tasks & Pipelines
+
+The default tasks and pipelines can be updated by forking the Kabanero Pipelines repo and editing the files under `pipelines/incubator`.  The easiest way to generate the archive to be used by Kabanero CRD is to drive the  link:https://github.com/kabanero-io/kabanero-pipelines/blob/master/ci/package.sh[package.sh] script.  The script will generate the archive file with the necessary pipeline artifacts and a manifest.yaml describing the contents of the archive.  
```

Comment:
Change
to be used by Kabanero CRD is to drive
to
for use by the Kabanero CRD is to run

Change
The script will generate
to
The script generates

Change
describing
to
that describes